Dockerfile

```diff
@@ -17,8 +17,42 @@ RUN go build -ldflags "-X main.version=${VERSION}" -o auroraboot
 FROM fedora:$FEDORA_VERSION AS default
 RUN dnf -y update
 ## ISO+ Arm image + Netboot + cloud images Build depedencies
-RUN dnf in -y bc jq genisoimage docker sudo parted e2fsprogs erofs-utils binutils curl util-linux udev rsync \
-    dosfstools mtools xorriso lvm2 zstd sbsigntools squashfs-tools kpartx grub2 openssl
+RUN dnf in -y bc \
+              binutils \
+              curl \
+              docker \
+              dosfstools \
+              e2fsprogs \
+              erofs-utils \
+              gdisk \
+              genisoimage \
+              git \
+              grub2 \
+              jq \
+              kpartx \
+              lvm2 \
+              mtools \
+              openssl \
+              parted \
+              qemu-img \
+              qemu-system-x86 \
+              qemu-tools \
+              rsync \
+              sbsigntools \
+              squashfs-tools \
+              sudo \
+              udev \
+              util-linux \
+              xfsprogs \
+              xorriso \
+              zstd \
+              python3-cryptography python3-pefile # ukify deps
+
+# systemd-ukify systemd-boot
+# Install grub2-efi-x64 only on x86 arches
+RUN if [ "$(uname -m)" == "x86_64" ]; then dnf install -y grub2-efi-x64; fi
+# Install grub2-efi-arm64 only on arm64 arches
+RUN if [ "$(uname -m)" == "aarch64" ]; then dnf install -y grub2-efi-aa64; fi
```

Comment:
Does this work when building multi arch images? I mean, does `uname` report the correct thing?

Reply:
aparently these are not needed anymore, and for finding the arch I think we actually use this: https://github.com/kairos-io/AuroraBoot/blob/main/pkg/utils/common.go#L334